integration_tests/integration/recentArrivals/recentArrivals.cy.ts

```diff
@@ -108,4 +114,72 @@ context('A user can view all recent arrivals', () => {
     recentArrivalsSearchPage.clearSearch().click()
     Page.verifyOnPage(RecentArrivalsPage)
   })
+
+  it('Should display prisoner summary page', () => {
+    cy.task('stubGetBodyScan', {
+      prisonNumber: recentArrival.prisonNumber,
+      details: {
+        prisonNumber: recentArrival.prisonNumber,
+        bodyScanStatus: 'OK_TO_SCAN',
+        numberOfBodyScans: 1,
+        numberOfBodyScansRemaining: 100,
+      },
+    })
+
+    cy.signIn()
+    const recentArrivalsPage = RecentArrivalsPage.goTo()
+    recentArrivalsPage.recentArrivals(1, today).name().click()
+
+    const prisonerSummaryPage = new PrisonerSummaryPage(`${recentArrival.lastName}, ${recentArrival.firstName}`, {
+      hasBackLink: false,
```

Comment:
don't need hasBackLink as a parameter either do you?

Reply:
agreed